rest-mvc/src/test/java/com/intuit/tank/rest/mvc/rest/controllers/ScriptControllerTest.java

```diff
@@ -181,6 +181,18 @@ public void testUploadScript() throws IOException {
         verify(scriptService).uploadProxyScript("testName", 0, "gzip", null);
     }
 
+    @Test
```

Comment:
@Zakaria-Kofiro Can we add one unit test error updating use-case or when a file does not exist?

Reply:
Yup, will add.